src/server/api/controllers/eventsNews.js

```diff
@@ -0,0 +1,184 @@
+import { badgerBrain } from '../../../shared/config';
+
+const badgerBrainEndpoint =
+  process.env.NODE_ENV === 'production'
+  ? badgerBrain.remote
+  : badgerBrain.local;
+
+const getRequestOptions = (req, body) => ({
+  method: 'POST',
+  headers: Object.assign({}, {
+    'Content-Type': 'application/graphql',
+  }, req.query.token && {
+    'X-Preview': req.query.token,
+  }),
+  body,
+});
+
+const basicFields = `
+  id
+  slug
+  tags
+  title
+  strapline
+  internalLinks {
+    title
+    url
+  }
+  externalLinks {
+    title
+    url
+  }
+  body {
+    type
+    text
+  }
+  featureImageFilename
+`;
+
+const dateTimeFields = `
+  iso
+  date
+  month
+  monthSym
+  year
+`;
+
+const dateTimeFieldsEvents = `
+  startDateTime {
+    ${dateTimeFields}
+  }
+  endDateTime {
+    ${dateTimeFields}
+  }
+`;
+
+const dateTimeFieldsNews = `
+  datetime {
+    ${dateTimeFields}
+  }
+`;
+
+const fullEventsQuery = `
+  ${basicFields}
+  ${dateTimeFieldsEvents}
+`;
+
+const fullNewsQuery = `
+  ${basicFields}
+  ${dateTimeFieldsNews}
+`;
+
+const sortNews = (list) =>
+  list.sort((a, b) =>
+    new Date(b.datetime.iso) - new Date(a.datetime.iso));
+
+const sortEvents = (list) =>
+  list.sort((a, b) =>
+    new Date(b.startDateTime.iso) - new Date(a.startDateTime.iso));
+
+export function getNewsItem(req, res) {
+  const body = `
+    query {
+      news(id: "${req.params.id}") {
+        ${fullNewsQuery}
+      }
+    }
+  `;
+
+  fetch(badgerBrainEndpoint, getRequestOptions(req, body))
+    .then((response) => {
+      response.json().then((json) => {
```

Comment:
Below we return the `response.json()` call and then chain another `then` from there, we could change this to be consistent? I think it's a tiny but more readable.